docs/example-scenario/apps/apim-api-scenario.md

```diff
@@ -1,64 +1,62 @@
+1. The existing on-premises web application will continue to directly consume the existing on-premises web services.
+2. Calls from the existing web app to the existing HTTP services will remain unchanged. These calls are interal to the corporate network.
+3. Inbound calls are made from Azure to the existing internal services:
+    * The security team allows traffic from the APIM instance to pass through the corporate firewall to the existing on-premises services [using secure transport (HTTPS/SSL)][apim-ssl].
+    * The operations team will allow inbound calls to the services only from the APIM instance. This requirement is met by [white-listing the IP address of the APIM instance][apim-whitelist-ip] within the corporate network perimeter.
+    * A new module is configured into the on-premises HTTP services request pipeline (to act upon **only** those connections originating externally), which will validate [a certificate which APIM will provide][apim-mutualcert-auth].
+1. The new API:
+    * Is surfaced only through the APIM instance, which will provide the API facade. The new API won't be accessed directly.
     * Is developed and published as an [Azure PaaS Web API App][azure-api-apps].
-    * Is white-listed (via [Web App settings][azure-appservice-ip-restrict]) to accept only the [API-M VIP][apim-faq-vip].
+    * Is white-listed (via [Web App settings][azure-appservice-ip-restrict]) to accept only the [APIM VIP][apim-faq-vip].
     * Is hosted in Azure Web Apps with Secure Transport/SSL turned on.
-    * Has Authorization turned on, [provided by the Azure App Service][azure-appservice-auth] using Azure Active Directory and OAuth2.
-5. The new browser-based Web Application will depend on the Azure API-Management instance for *both* the existing HTTP API and the new API.
+    * Has authorization enabled, [provided by the Azure App Service][azure-appservice-auth] using Azure Active Directory and OAuth2.
+2. The new browser-based web application will depend on the Azure API Management instance for **both** the existing HTTP API and the new API.
 
-The API-M instance will be configured to map the legacy HTTP services to a new API contract. By doing this, the new Web UI is unaware it's integrating with a set of legacy services/APIs and new APIs. In the future, the project team plans to gradually port functionality to the new APIs and retire the original services. These changes will be handled within API-M configuration leaving the front-end UI unaffected and avoiding redevelopment work.
+The APIM instance will be configured to map the legacy HTTP services to a new API contract. By doing this, the new Web UI is unaware of the integration with a set of legacy services/APIs and new APIs. In the future, the project team will gradually port functionality to the new APIs and retire the original services. These changes will be handled within APIM configuration, leaving the front-end UI unaffected and avoiding redevelopment work.
```

Comment:
Is this an example of [Strangler pattern](https://docs.microsoft.com/en-us/azure/architecture/patterns/strangler)?

Reply:
I've never heard this term before and our docs are the top Google hit for it.  Since it's not widely used so I'd like to avoid it here.